src/test/java/org/folio/rest/impl/EntitiesCrudBasicsTest.java

```diff
@@ -104,6 +106,26 @@
     return transactionEntities.stream().filter(e -> !e.equals(TestEntities.ORDER_TRANSACTION_SUMMARY));
   }
 
+  @Test
+  public void testPostRecordEmptySeriesFY() throws IOException {
+    logger.info("=== Test create {} record empty series for FY and it should calculate series ===", TestEntities.FISCAL_YEAR);
+    JsonObject record = TestEntities.FISCAL_YEAR.getMockObject();
+    record.putNull("series");
+    
+    FiscalYear fiscalYear = verifyPostResponse(TestEntities.FISCAL_YEAR.getEndpoint(), record, APPLICATION_JSON, CREATED.getStatusCode()).as(FiscalYear.class);
+    assertThat(fiscalYear.getSeries(), is(notNullValue()));
+  }
+
+  @Test
+  public void testUpdateRecordEmptySeriesFY() {
+    logger.info("=== Test update {} record with empty series for FY and it should calculate series ===", TestEntities.FISCAL_YEAR);
+
+    JsonObject body = TestEntities.FISCAL_YEAR.getMockObject();
+    body.putNull("series");
+
+    verifyPut(TestEntities.FISCAL_YEAR.getEndpointWithId((String) body.remove(ID)), body, "", NO_CONTENT.getStatusCode());
+  }
```

Comment:
Please add a check to assert that the record sent to storage had the series populated

Reply:
Done